index.html

```diff
@@ -102,6 +101,10 @@ <h2 id="deliverables">Deliverables</h2>
               Timeline</a></dt>
           <dd>This specification defines an unified interface to store and
             retrieve performance metric data.</dd>
+          <dt>Performance Observer</dt>
```

Comment:
FWIW, the PerfObserver interface is part of Perf Timeline, so we probably don't need this entry.